src/prefect/client/orchestration.py

```diff
@@ -174,6 +182,13 @@ def get_client(httpx_settings: Optional[dict] = None) -> "PrefectClient":
     async with get_client() as client:
         await client.hello()
     ```
+
+    To return a synchronous client, pass sync_client=True:
+
+    ```python
+    with get_client(sync_client=True) as client:
+        client.hello()
+    ```
```

Comment:
Should we mention that the sync client has fewer methods than the async client? I foresee some users getting excited about a sync API client and then disappointed when it doesn't support all the methods that the async client does.

Reply:
Yea, once we finalize the sync implementation we should do that